src/app/home-page/recent-item-list/recent-item-list.component.ts

```diff
@@ -0,0 +1,74 @@
+import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core';
+import { PaginatedSearchOptions } from '../../shared/search/models/paginated-search-options.model';
+import { fadeIn, fadeInOut } from '../../shared/animations/fade';
+import { RemoteData } from '../../core/data/remote-data';
+import { PaginatedList } from '../../core/data/paginated-list.model';
+import { Item } from '../../core/shared/item.model';
+import { PaginationComponentOptions } from '../../shared/pagination/pagination-component-options.model';
+import { PaginationService } from '../../core/pagination/pagination.service';
+import { SearchService } from '../../core/shared/search/search.service';
+import { SortDirection, SortOptions } from '../../core/cache/models/sort-options.model';
+import { DefaultAppConfig } from '../../../config/default-app-config';
+import { AppConfig } from '../../../config/app-config.interface';
+import { ViewMode } from '../../core/shared/view-mode.model';
+import { SearchConfigurationService } from '../../core/shared/search/search-configuration.service';
+import {
+  toDSpaceObjectListRD
+} from '../../core/shared/operators';
+import {
+  Observable,
+} from 'rxjs';
+@Component({
+  selector: 'ds-recent-item-list',
+  templateUrl: './recent-item-list.component.html',
+  styleUrls: ['./recent-item-list.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  animations: [
+    fadeIn,
+    fadeInOut
+  ]
+})
+export class RecentItemListComponent implements OnInit {
+  itemRD$: Observable<RemoteData<PaginatedList<Item>>>;
+  paginationConfig: PaginationComponentOptions;
+  sortConfig: SortOptions;
+  appConfig: AppConfig = new DefaultAppConfig();
```

Comment:
Remove, don't use `DefaultAppConfig`, not overridable with yml files. Use `environment` (`import { environment } from '../../../environments/environment';`) instead.

Reply:
emove, don't use DefaultAppConfig, not overridable with yml files